lib/src/main/kotlin/net/dean/jraw/references/SubmissionReference.kt

```diff
@@ -82,6 +82,50 @@ class SubmissionReference internal constructor(reddit: RedditClient, id: String)
         }
     }
 
+    // TODO - Requires mod privileges for the subreddit that contains the post.
+    /**
+     * Sets the [spoilerStatus] of a post using the official Reddit spoiler flair.
+     */
+    @EndpointImplementation(Endpoint.POST_SPOILER, Endpoint.POST_UNSPOILER)
```

Comment:
Make sure to run the `:meta:endpoints` task to mark these endpoints as implemented (see [here](https://github.com/mattbdean/JRAW/tree/master/meta) for more).

Reply:
I included the updated .md files and an updated endpoint enum.  I didn't include `Version.java`, let me know if that was incorrect.

Reply:
You're good, that should only change when we change the version in Gradle